README.md

```diff
@@ -49,13 +48,16 @@ native dependencies.
 
 A script is provided to build Conda with dependencies:
 
-```
+```Bash
     # Create environment in ~/conda directory
     ./deployment/create-conda-environment.sh ~/conda
 
     # Activate the environment in the current shell
     . ~/conda/bin/activate
 
+    # Ensure build dependencies are installed
+    pip install "setuptools_scm[toml]>=6.2,<8"
```

Comment:
Ooops, forgot to flag that the docs hard code a dependency version. I can leave it as is, or change the comment to encourage users to grab the version string from `pyproject.toml`.

Any thoughts?